src/state/shellscripts/bwrap.sh

```diff
@@ -125,6 +125,11 @@ case "$COMMAND" in
         exit 2
 esac
 
+if ! command -v "$1" >/dev/null; then
+    echo "[ERROR] Command not found: $1" >&2
```

Comment:
Wondering, `echo -e` is not portable?

Reply:
We can leave that to another PR. It’s the way echo is used everywhere else in the sandbox scripts

Reply:
`-e` is not POSIX complient: https://www.unix.com/man-page/posix/1posix/echo/